src/components/SiteSearch.vue

```diff
@@ -212,8 +212,15 @@ export default class SiteSearch extends Vue {
 @import "./../styles/Imports";
 
 .s-sitesearch__input {
+  margin: 0;
   border: none;
-  padding: none;
+  height: 39px;
```

Comment:
should this be 40? or 38 if you're accounting border?

Reply:
yes, im accounting for 
border, it just looks better at 39 than 38